docs/user_guide/assessment.rst

```diff
@@ -6,6 +6,124 @@ Metrics
+the input data) and predicted values :math:`Y_{pred}` (by applying the model
+to the input data), and use these to compute a measure. For example, the
+*recall* or *true positive rate* is given by
+
+.. math::
+
+   P( Y_{pred}=1 \given Y_{true}=1 )
+
+That is, a measure of whether the model finds all the positive cases in the
+input data. The `scikit-learn` package implements this in
+:py:func:`sklearn.metrics.recall_score`.
+
+Suppose we have the following data we can see that the prediction is `1` in five
+of the ten cases where the true value is `1`, so we expect the recall to be 0.5:
+
+    >>> import sklearn.metrics as skm
+    >>> Y_true = [0, 1, 1, 1, 1, 0, 1, 0, 1, 0, 0, 0, 1, 1, 1, 1]
+    >>> Y_pred = [0, 0, 1, 0, 1, 1, 1, 0, 0, 1, 1, 1, 1, 0, 0, 1]
+    >>> skm.recall_score(Y_true, Y_pred)
+    0.5
+
+Metrics with Grouping
+^^^^^^^^^^^^^^^^^^^^^
+
+When considering fairness, each row of input data will have an associated
+group label :math:`g \in G`, and we will want to know how the metric behaves
+for each :math:`g`. To help with this, Fairlearn provides wrappers, which take
+an existing (ungrouped) metric function, and apply it to each group within a
+set of data.
+
+Suppose in addition to the :math:`Y_{true}` and :math:`Y_{pred}` above, we had
+the following set of labels:
+
+    >>> import pandas as pd
+    >>> group_membership_data = ['d', 'a', 'c', 'b', 'b', 'c', 'c', 'c', 'b', 'd', 'c', 'a', 'b', 'd', 'c', 'c']
+    >>> pd.DataFrame({ 'Y_true': Y_true, 'Y_pred': Y_pred, 'group_membership_data': group_membership_data})
+    Y_true  Y_pred group_membership_data
+    0        0       0                     d
+    1        1       0                     a
+    2        1       1                     c
+    3        1       0                     b
+    4        1       1                     b
+    5        0       1                     c
+    6        1       1                     c
+    7        0       0                     c
+    8        1       0                     b
+    9        0       1                     d
+    10       0       1                     c
+    11       0       1                     a
+    12       1       1                     b
+    13       1       0                     d
+    14       1       0                     c
+    15       1       1                     c
+
+    >>> import fairlearn.metrics as flm
+    >>> group_metrics = flm.group_summary(skm.recall_score, Y_true, Y_pred, sensitive_features=group_membership_data, sample_weight=None)
+    >>> print("Overall recall = ", group_metrics.overall)
+    Overall recall =  0.5
+    >>> print("recall by groups = ", group_metrics.by_group)
+    recall by groups =  {'a': 0.0, 'b': 0.5, 'c': 0.75, 'd': 0.0}
+
+Note that the overall recall is the same as that calculated above in the
+Ungrouped Metric section, while the `by_group` dictionary matches the values
+we calculated by inspection from the table above.
```

Comment:
'we calculated by inspection' .... I don't see this happening. Did I mess up in the original notebook?

Reply:
What do you mean by "I don't see this happening"? Did you spot an error or do you mean I copy-pasted something badly? I checked some of the numbers but they look fine to me

Reply:
Sounds like I messed up the original notebook. There's no prior calculation of the by group values above that I see.